packages/ignite/src/plugins/default/commands/new.js

```diff
@@ -29,35 +29,51 @@ module.exports = async function (context) {
     return
   }
 
+  // Install the ignite-* packages from the local dev directory.
+  //
+  // This is pretty much always what we while we dev.
+  //
+  // To test what live is like, you can run `ignite new FooTown --live`.
+  //
+  // TODO(steve): Don't forget to remove this when we launch... open to better ways of handling it.
+  const igniteDevPackagePrefix = parameters.options.live || `${__dirname}/../../../../../ignite-`
+
   // First we ask!
   const answers = await context.prompt.ask(installWalkthrough)
   // then we kick off (TODO: Would be awesome to have this kick off during questions)
   // we need to lock the RN version here
   info('Creating new RN project')
   // TODO make sure `react-antive --version has react-native-cli 2.x otherwise failure`
-  await system.run(`react-native init ${projectName} --version 0.38.0`)
+  const reactNativeVersion = '0.40.0'
+  await system.run(`react-native init ${projectName} --version ${reactNativeVersion}`)
+
+  // switch to the newly created project directory to continue the rest of these commands
   process.chdir(projectName)
+
   info('Add ignite basic structure with unholy')
-  await system.run(`ignite add basic-structure ${projectName} --unholy`)
+  await system.run(`ignite add ${igniteDevPackagePrefix}basic-structure ${projectName} --unholy`)
+
   info('Install all those unholy goodies')
   await system.run('yarn || npm i')
+
   // info('Link up all those unholy goodies')
   // the following never returns - without await it keeps the shell forever!
   system.run('react-native link &')
+
   info('Add ignite basic generators')
-  await system.run('ignite add basic-generators')
+  await system.run(`ignite add ${igniteDevPackagePrefix}basic-generators`)
 
   // now run install of Ignite Plugins
   if (answers['dev-screens']) {
-    await system.run('ignite add dev-screens')
+    await system.run(`ignite add ${igniteDevPackagePrefix}dev-screens`)
   }
 
   if (answers['vector-icons'] === 'react-native-vector-icons') {
-    await system.run('ignite add vector-icons')
+    await system.run(`ignite add ${igniteDevPackagePrefix}vector-icons`)
   }
 
   debug(answers)
 
-  info('cd to your new project')
+  info(`Please type cd ${projectName}`)
   info('YOU MUST NOW RUN `react-native link` Sorry we are not doing that for you yet')
```

Comment:
For the record, you're going to hell for writing 61.

But yes, I'll remove this message.  And join you in hell for aiding & abetting.

🚔